mockfirestore/main.py

```diff
@@ -66,6 +67,11 @@ class DocumentSnapshot:
     def __init__(self, reference: 'DocumentReference', data: Document) -> None:
         self.reference = reference
         self._doc = deepcopy(data)
+        self._id = ''.join(random.choices(ALPHABET, k=10))
+
+    @property
+    def id(self) -> str:
+        return self._id
```

Comment:
The `id` property was added in #11 , so it's not needed here.

Reply:
I’ll rebased and remove it.